packages/block-editor/src/components/block-list/style.scss

```diff
@@ -378,10 +378,25 @@
 }
 
 // Sibling inserter / "inbetweenserter".
-// Fade it in after a delay.
+.block-editor-block-list__empty-block-inserter,
+.block-editor-default-block-appender,
 .block-editor-block-list__insertion-point-inserter,
 .block-editor-block-list__block-popover-inserter {
 	.block-editor-inserter__toggle.components-button {
+		// Basic look
+		background: $dark-gray-primary;
+		border-radius: $radius-block-ui;
+		color: $white;
+
+		// Special dimensions for this button.
+		min-width: 24px;
+		height: 24px;
+
+		&:hover {
+			color: $white;
+		}
```

Comment:
This style is good but I wonder if this should be a button style variation (a prop in the Button component)

Reply:
I think it should be. For example, we need a dark button style for a future Links iteration:

<img width="567" alt="Screenshot 2020-03-06 at 11 32 19" src="https://user-images.githubusercontent.com/1204802/76075947-31036900-5f9e-11ea-8369-37d917ac4367.png">

Per [this comment](https://github.com/WordPress/gutenberg/pull/20441#issuecomment-591400883), we should also consider expanding the documentation to add more nuance than just "primary, secondary, tertiary" and so on. While there is certainly a visual hierarchy that an be used in the toolset of UI design, the markup is the same underneath, so it might make sense to look at variations than just an absolute hierarchy. 

So perhaps a bigger undertaking than just adding a button style?

Reply:
Not sure I understand, Technically hierarchy or variation seems like the same thing for me. It's just a matter of finding a name for it.

Design-wise, there needs to be some guidelines maybe (when to use what).

Reply:
Right — but I mean you can have a primary black, primary blue, primary red, potentially. Just mean that we should not have a fourth level in the hierarchy.

Reply:
Got it, you're right, that's more "theme support"

Reply:
Yes but to be clear, I meant that even in the default style, the primary buttons are theme colored, AND the URL buttons are black, i.e. they coexist. And the red might exist as well, as a destructive action.

Reply:
In that case, they seem more like variations to me.

`primary, secondary, tertiary, error` and the black one I can't name it.

Reply:
pr `primary.hasError` `secondary.hasError`...

Reply:
Not something we have to decide right now, but it's something I know @mtias and @pablohoneyhoney have thought about.